lab01/sections/intro.tex

```diff
@@ -1,3 +1,52 @@
 \section{Introduction}\label{sec:intro}
-Short section on the background and motivation i.e. what the experiment is about and what is being measured.
-References are appreciated \cite{lab-manual}.
+This experiment will develop a model for a DC motor controller.
+The angular velocity of a DC motor in an armature configuration, shown in Figure~\ref{fig:motorschematic}, can be approximated by a first order system.
+We can vary the applied voltage $u_m$ in a static configuration to determine the internal resistance of the particular motor $R_m$ and its torque constant $k_m$.
+These two values let us determine the the gain $K$ and time constant $\tau$ of the governing first order equation
+\begin{equation}\label{eq:tf}
+  G(s) = {K \over \tau s + 1}.
+\end{equation}
+This model will be improved by analyzing the system input and output under dynamic conditions.
+Measurements in dynamic conditions make fewer assumptions about the system and give a better approximation for $K$ and $\tau$.
```

Comment:
In the draft the tau is written as "tau", and in the previous paragraph it is the symbol for tau. These should be the same, as the symbol, for consistency in my opinion.